src/goto-instrument/contracts/dynamic-frames/dfcc_contract_functions.cpp

```diff
@@ -66,6 +69,29 @@ dfcc_contract_functionst::dfcc_contract_functionst(
 
   spec_functions.to_spec_frees_function(
     spec_frees_function_id, nof_frees_targets);
+
+  // instrument the generated functions for side-effect checking
+  std::set<irep_idt> function_pointer_contracts;
```

Comment:
I don't quite understand how the comment fits with the declaration.

Reply:
this comment applies to the two next lines of code.

Reply:
deleted